test/app/components/callbacks_component.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class CallbacksComponent < ActionView::Component::Base
+  before_render :before
+  around_render :around
+  after_render  :after
+
+  attr_accessor :events
+
+  def initialize(events: [])
+    @events = events
+  end
```

Comment:
I know its just test code but why allow events to be passed in 
```suggestion
 def initialize()
    @events = []
  end
```

Reply:
ok